### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: __unported__/crm_claim_ext/__openerp__.py [hunk 1]
```diff
@@ -37,6 +37,7 @@
     """,
     'author': 'esamyn',
     'website': 'http://www.erp-236.com',
+    'license': 'AGPL-3',
```
Comment (yvaucher) [inline __unported__/crm_claim_ext/__openerp__.py]: Change to GPL

**Comment by lepistone** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by max3903** [comment]
:+1:

**Pull request merged**